kong/init.lua

```diff
@@ -329,6 +329,20 @@ function Kong.init()
   assert(runloop.build_api_router(dao, "init"))
 end
 
+local function list_migrations(migtable)
+  local list = {}
+  for _, t in ipairs(migtable) do
+    local mignames = {}
+    for _, mig in ipairs(t.migrations) do
+      table.insert(mignames, mig.name)
+    end
+    table.insert(list, string.format("%s (%s)", t.subsystem,
+                       table.concat(mignames, ", ")))
+  end
+  return table.concat(list, ", ")
```

Comment:
A blank separator (`" "`) was the correct separator here (since `,` is used to separate migrations within the parenthesis). Only one of the two previous logging logics got it right :sweat_smile:

Reply:
But doesn't `foo (hello, world), bar (baz, bla), ble (bli, blo, blu)` actually look nicer? :)

Reply:
à chacun son goût! ;) all right a space it is